tests/orquestra/integrations/qiskit/backend/backend_test.py

```diff
@@ -156,10 +156,11 @@ def test_aggregate_measurements_extracts_correct_qubits(
         )
 
         # Then
-        if backend_with_readout_correction.noise_inversion_method == "pseudo_inverse":
-            assert results[0].get_counts() == {"0000": 1, "1001": 48, "1101": 1}
-        else:
-            assert results[0].get_counts() == {"1001": 48}
+        lst = backend_with_readout_correction.list_virtual_to_physical_qubits_dict
+        assert len(lst) == 1
+        assert lst[0] != {0: 0, 1: 1, 2: 2, 3: 3, 4: 4}
```

Comment:
This is a weird assertion?

Reply:
Added a comment to the most recent version explaining the assertion.

Reply:
Now it totally makes sense!